bb-core/src/main/java/com/cognifide/qa/bb/loadable/hierarchy/WebElementInterceptor.java

```diff
@@ -1,41 +1,43 @@
 /*
  * Copyright 2016 Cognifide Ltd..
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
  *
- *      http://www.apache.org/licenses/LICENSE-2.0
+ * http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
  */
 package com.cognifide.qa.bb.loadable.hierarchy;
 
-import com.cognifide.qa.bb.loadable.context.ClassFieldContext;
-import com.cognifide.qa.bb.loadable.annotation.LoadableComponent;
-import com.cognifide.qa.bb.loadable.context.ConditionContext;
-import com.cognifide.qa.bb.qualifier.PageObject;
-import com.cognifide.qa.bb.utils.AopUtil;
-import com.cognifide.qa.bb.webelement.BobcatWebElement;
-import com.google.inject.Inject;
-
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
+import java.util.Set;
 
+import com.cognifide.qa.bb.exceptions.BobcatRuntimeException;
+import com.cognifide.qa.bb.loadable.LoadableProcessorFilter;
 import org.aopalliance.intercept.MethodInterceptor;
 import org.aopalliance.intercept.MethodInvocation;
 import org.openqa.selenium.WebElement;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.cognifide.qa.bb.loadable.annotation.LoadableComponent;
+import com.cognifide.qa.bb.loadable.context.ClassFieldContext;
+import com.cognifide.qa.bb.loadable.context.ConditionContext;
+import com.cognifide.qa.bb.qualifier.PageObject;
+import com.cognifide.qa.bb.utils.AopUtil;
+import com.cognifide.qa.bb.webelement.BobcatWebElement;
+import com.google.inject.Inject;
+
 /**
- * This intercepts invocation of {@link WebElement} methods and runs evaluation of conditions provided in the
```

Comment:
Please restore formatting for this JAVADOC.
Formatting changes should be done as separate pull request
(should have)